torch/_subclasses/fake_tensor.py

```diff
@@ -2514,21 +2515,61 @@ def wrap(e: T) -> Union[T, FakeTensor]:
 
         return tree_map(wrap, r)
 
-    def create_symbolic_nested_int(
-        self, *, nt_tensor_id: Optional[int] = None
+    def get_nested_int(
```

Comment:
will leave all these fake tensor changes to @ezyang to review.

I don't have much preference here except that I feel like nested int specific stuff should be absolutely minimized within fake tensor / fake-ification land. I'll leave it to you two to determine what that looks like though